### PR TITLE
Campgrounds

### PR DESCRIPTION
**Pull request by schoure98**
Hello, I have added Campgrounds component which contains state wise number of campgrounds available. The data is coming from /campgrounds end point from National park services.
This visualization is created to understand the statistical data of campgrounds in US states. 

Attached below is the expected screenshot. 

Please let me know your feedback on coding suggestions or best practices/ design suggestions. 

Screenshot 2023-05-28 at 9 03 40 AM

### CODE REVIEW

File: park-dashboard/src/Components/Campgrounds/CampgroundsChart.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    const stateCode = camp.address;
+    acc[stateCode] = (acc[stateCode] || 0) + 1;
+    return acc;
+  }, {});
+
+  const filteredCounts = Object.entries(counts).filter(
+    ([state, count]) => count >= 10
+  );
+
+  const stateCodes = filteredCounts.map(([state, count]) => state);
```
Comment (juhiaugustine) [inline park-dashboard/src/Components/Campgrounds/CampgroundsChart.jsx:31]: Did you try using Object.keys and Object.values here instead of map function?

**Comment by juhiaugustine** [comment]
Overall looks good 👍

File: park-dashboard/src/Components/Campgrounds/Campgrounds.js
Comment (rutujapadgilwar) [inline park-dashboard/src/Components/Campgrounds/Campgrounds.js:1]: Great work on segregating the file for loading campground data. I plan to utilize the same file for my upcoming work as well. Could you please rename the file to "LoadCampgroundData.js" for better naming conventions?

File: park-dashboard/src/Components/Campgrounds/CampgroundsChart.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          "rgba(50, 80, 150, 0.8)",
+          "rgba(220, 120, 180, 0.8)",
+          "rgba(80, 220, 120, 0.8)",
+          "rgba(180, 80, 220, 0.8)",
+          "rgba(120, 180, 80, 0.8)",
+          "rgba(230, 160, 60, 0.8)",
+          "rgba(60, 230, 160, 0.8)",
+          "rgba(50, 150, 50, 0.8)",
+        ],
+        borderWidth: 1,
```
Comment (rutujapadgilwar) [inline park-dashboard/src/Components/Campgrounds/CampgroundsChart.jsx:76]: For a more cohesive and visually appealing look, you can add the border colors to match the background colors, eliminating the presence of white lines

File: park-dashboard/src/LoadParkData.js [hunk 3]
```diff
@@ -23,5 +23,6 @@ export const LoadParkData = () => {
     fetchData();
   }, []);
 
+  console.log(parkData);
```
Comment (rutujapadgilwar) [inline park-dashboard/src/LoadParkData.js]: Remove the unwanted console.log

**Review by rutujapadgilwar — CHANGES_REQUESTED** [review]
I suggested small changes overall the chart looks nice.

File: park-dashboard/src/Components/Campgrounds/CampgroundsChart.jsx (hunk 1 above)
Comment (schoure98) [reply park-dashboard/src/Components/Campgrounds/CampgroundsChart.jsx:31]: Hey Juhi, filteredCounts is an array that is why I am using map here and not able to use Object.keys and values.

File: park-dashboard/src/LoadParkData.js (hunk 3 above)
Comment (schoure98) [reply park-dashboard/src/LoadParkData.js]: Removed

File: park-dashboard/src/Components/Campgrounds/CampgroundsChart.jsx (hunk 2 above)
Comment (schoure98) [reply park-dashboard/src/Components/Campgrounds/CampgroundsChart.jsx:76]: Thanks for suggesting this, I have added the border colors.

File: park-dashboard/src/Components/Campgrounds/Campgrounds.js
Comment (schoure98) [reply park-dashboard/src/Components/Campgrounds/Campgrounds.js:1]: Sure, I have changed the file name now.

**Pull request merged**